src/annotator/anchoring/text-range.ts

```diff
@@ -76,8 +76,14 @@ function resolveOffsets(element, ...offsets) {
   return results;
 }
 
-export let RESOLVE_FORWARDS = 1;
-export let RESOLVE_BACKWARDS = 2;
+/**
+ * When resolving a TextPosition, specifies the direction to search for the
+ * nearest text node if `offset` is `0` and the element has no text.
```

Comment:
I think we'll be able to generalize this description in future - eg. to something like "which direction to traverse the DOM in when resolving an offset that could be resolved to one of several positions."